app/models/comment.rb

```diff
@@ -1,4 +1,9 @@
 class Comment < ApplicationRecord
   belongs_to :author, class_name: 'User'
   belongs_to :post
+  after_save :update_comments_counter
+
+  def update_comments_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
- **Suggestion**:
- Make `update_comments_counter` **private method** as I commented in Post.

Reply:
- Well resolved 👍🏽